TrailBlaze/View Controllers/HomeViewController.m

```diff
@@ -235,11 +263,11 @@ - (void) setUpHomeViewForRerunStart {
     //placeholder run id
     [Run retreiveSpecificRunObject:@"dsfadkg" completion:^(PFObject * _Nonnull runObject, NSError * _Nullable err) {
         if (runObject) {
-            PaceImprovementTracker *pacer = [[PaceImprovementTracker alloc] initWithRunID:@"dfsadfsg"];
-            pacer.bestPacesDictionary = runObject[@"pacesDictionary"];
-            pacer.polylinePoints = [Utils jsonStringToArray:runObject[@"polylineCoords"]];
-            if ([PaceImprovementTracker isAtStartPosition:self->currentLocation firstPoint:pacer.polylinePoints[0]]) {
-                [pacer paceTracker:pacer.polylinePoints userLocation:self->currentLocation bestPaces:pacer.bestPacesDictionary];
+            self->pacer.bestPacesDictionary = runObject[@"pacesDictionary"];
+            self->pacer.polylinePoints = [Utils jsonStringToArray:runObject[@"polylineCoords"]];
```

Comment:
if `bestPacesDictionary` and `polylinePoints` are going to be private, we could add a `[pacer configureWithRunObject:runObject]` api to inject these values

Reply:
Yh I added them to the current initializer which already take the runObject as a parameter